scripts/pre-commit.sh

```diff
@@ -132,7 +132,7 @@ echo "Downloading e2e test dependencies"
 ./scripts/download-e2e-binaries.sh
 
 CREATE_INSECURE_REGISTRY=y CONFIGURE_INSECURE_REGISTRY=y OVERWRITE_KUBECONFIG=y \
-    ./scripts/create-clusters.sh
+    KIND_TAG="v1.14.0" ./scripts/create-clusters.sh
```

Comment:
(no action required) Looks like `v1.14.1` is available.

Reply:
Nice!  It wasn't there when I looked earlier today.